src/main/kotlin/snyk/amplitude/api/ExperimentUser.kt

```diff
@@ -0,0 +1,15 @@
+package snyk.amplitude.api
+
+import com.google.gson.annotations.SerializedName
+
+class ExperimentUser(
```

Comment:
data class?

Reply:
The purpose of this class is not to hold data only, but handle changing userId (atm not implemented).